java/src/main/native/src/ColumnViewJni.cpp

```diff
@@ -1407,6 +1411,10 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnView_binaryOpVS(JNIEnv *env, j
         out->set_null_mask(std::move(new_mask), new_null_count);
       }
 
+      if (cudf::may_have_nonempty_nulls(*out)) {
```

Comment:
Same comment here. This is not needed because it is only working on fixed width types.

Reply:
Thank you! Changes updated.